x-pack/plugin/mapper-unsigned-long/src/yamlRestTest/resources/rest-api-spec/test/50_script_values.yml

```diff
@@ -196,3 +199,36 @@ setup:
                 source: "Math.abs(doc['ul'].value)"
 
   - match: { hits.total.value: 5 }
+
+---
+"Scripted Metric":
+  - skip:
+      version: " - 7.99.99"
+      reason: "context for scripted metric was added in 8.0"
+
+  - do:
+      search:
+        index: test1
+        body:
+          query:
+            term:
+              ul: 18446744073709551615
+          aggs:
+            sqrt:
+              scripted_metric:
+                init_script: |
+                  state.sqrd = []
+                map_script: |
+                  double v = field('ul').asBigInteger(BigInteger.ZERO).doubleValue();
```

Comment:
This is added in 7.16

Reply:
@stu-elastic Thanks for your feedback. Indeed `field` API was added in 7.16, but `scripted_metric` agg context was added to `unsigned_long` only from 8.1 ( I should correct `8.0` to `8.1`).